_docs/administration/user-settings.md

```diff
@@ -1,22 +1,75 @@
 
-You can configure your user settings in the [dedicated area](https://g.codefresh.io/2.0/user-settings){:target="\_blank"}
+If OAuth2 authorization has been set up by your admin, you can authorize GitHub access using OAuth2.  
+You can always generate a personal access token from GitHub and then add the same.
+ 
+> You can use the same token for multiple runtimes, if you need access to more than one runtime. 
+You must however authorize GitHub access or add the personal access token for each runtime.
 
-{% include
-image.html
-lightbox="true"
-file="/images/administration/user-settings/main.png"
-url="/images/administration/user-settings/main.png"
-alt="Invite users"
-caption="Invite users"
-max-width="100%"
-%}
 
-For the time being the possible configuration and usability of the screen is the same as classic Codefresh and you can find the full documentation in the [classic Codefresh Docs](https://codefresh.io/docs/docs/administration/user-settings/){:target="\_blank"}
+
+### Git personal token permissions
+For OAuth2, the permissions and expiry date are pre-configured by the administrator.  
+For Git personal access tokens, make sure you select `repo` scope for commits and other actions when you generate the token.
+
+{% include 
+   image.html 
+   lightbox="true" 
+   file="/images/getting-started/github-pat.png" 
+   url="/images/getting-started/github-pat.png" 
+   alt="Permissions for Git personal token" 
+   caption="Permissions for Git personal token"
+   max-width="30%" 
+   %}
+
+### How to update a Git personal token
+Update your Git personal access token for each runtime when needed.  
+To authorize with OAuth2, you have to supply your credentials, and when authorized are automatically directed to the Git Personal Tokens page. 
+
+
+**Before you begin**  
+* For a Git personal access tokens, generate a valid personal access token from your Git provider  
+
+**How to**  
+
+1. In the CSDP UI, go to [User Settings](https://g.codefresh.io/2.0/user-settings){:target="\_blank"}.
+1. Select **+Add Token**. 
+1. For OAuth2:
+   * In the Add Token panel, select **Authorize Access to GitHub**.
+    > If the application is not registered, you get a _Git app not registered_ error. Contact your admin to continue.  
+   *  Enter your credentials, and select **Sign In**.
+
+      {% include 
+   image.html 
+   lightbox="true" 
+   file="/images/administration/user-settings/oauth-user-authentication.png" 
+   url="/images/administration/user-settings/oauth-user-authentication.png" 
+   alt="Authorizing access with OAuth2" 
+   caption="Authorizing access with OAuth2"
+   max-width="30%" 
+   %}
+  * Complete the verification, if you have two-factor authentication for example.
+  The token is generated, and you are automatically directed to the Git Personal Access Tokens page. 
+ 1. For Git personal access tokens: Paste the generated token in the **Token** field, and select **+Add Token**. 
```

Comment:
Wrong numbered list structure for options `OAuth2` and `personal access token`

Reply:
I changed the structure but known issue with lists in md